BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -590,6 +587,11 @@ - (void)writeDataToOutputStream
 {
     while ([self.outputStream hasSpaceAvailable])
     {
+        if (self.isCancelled) {
```

Comment:
Anywhere else to handle this or are these 2 the only stream methods?

How do we make sure this is handled correctly if/when another method is added?